### DISCUSSION TITLE
Will the @moddable/typings package be updated?

### DISCUSSION THREAD

**Discussion by stc1988** [Moddable-OpenSource/moddable#1362] [General]
I'm trying to use `mcpack` with Typescript, but [@moddable/typings](https://www.npmjs.com/package/@moddable/typings) seems to be a bit old.

Is there any plan to update the @moddable/typings package?

**Comment by phoddie** [comment]
> I'm trying to use mcpack...

Great!

> Is there any plan to update the @moddable/typings package?

Yes, we should be doing that as part of our regular release process. Obviously, that hasn't been working. I'll look into that.

**Reply by phoddie** [reply]
Just a quick note to acknowledge that we didn't get this done yet, but still intend to do so. Thank you for your patience.

**Reply by phoddie** [reply]
Version 1.0.1 of the Typings package should now be live. Please check to see that this is up-to-date with the latest Moddable SDK typings. Thank you.

**Reply by stc1988** [reply]
I haven’t checked the differences, but it looks like version 1.0.2 has been released. Should we use that instead?

**Reply by mkellner** [reply]
@stc1988 - I have built the latest typings from TOT to exercise the process. Please use 1.0.2 until there is a newer version.

**Reply by stc1988** [reply]
I try to run `examples/packages/ts-hello`, but `npm run build` failed.
It seems that [v1.0.2](https://www.npmjs.com/package/@moddable/typings/v/1.0.2?activeTab=code)   does not include these files.

```
npm run build

> @moddable/example-ts-hello@1.0.0 build
> tsc

error TS2688: Cannot find type definition file for './node_modules/@moddable/typings/mcpack'.
  The file is in the program because:
    Entry point of type library './node_modules/@moddable/typings/mcpack' specified in compilerOptions

  tsconfig.json:15:3
    15   "./node_modules/@moddable/typings/mcpack"
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    File is entry point of type library specified here.

error TS2688: Cannot find type definition file for './node_modules/@moddable/typings/xs'.
  The file is in the program because:
    Entry point of type library './node_modules/@moddable/typings/xs' specified in compilerOptions

  tsconfig.json:14:3
    14   "./node_modules/@moddable/typings/xs",
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    File is entry point of type library specified here.


Found 2 errors.
```

**Reply by mkellner** [reply]
Thank you for the report. I have removed 1.0.2 until we sort it out.

**Comment by phoddie** [comment]
We are now updating @moddable/typings as part of the Moddable SDK release process. The version number of the npm package matches the Moddable SDK version it is generated from (v5.0.0 as of this writing).
